src/main/java/org/jabref/logic/importer/util/MetaDataParser.java

```diff
@@ -60,12 +60,17 @@ public MetaData parse(MetaData metaData, Map<String, String> data, Character key
             } else if (entry.getKey().startsWith(MetaData.SELECTOR_META_PREFIX)) {
                 metaData.addContentSelector(ContentSelectors.parse(entry.getKey().substring(MetaData.SELECTOR_META_PREFIX.length()), StringUtil.unquote(entry.getValue(), MetaData.ESCAPE_CHARACTER)));
                 continue;
+            } else if (entry.getKey().startsWith(MetaData.FILE_DIRECTORY + 'l')) {
```

Comment:
The `l` stands for `latex` right? As the user usually does not have to type this by hand, I would prefer writting it out. Please also extract the key `MetaData.FILE_DIRECTORY + 'latex' to a new variable in `MetaData` (this class should collect all relevant data for parsing/reading of metadata).

The serialization format should be `fileDirectoryLatex-user`.